sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -501,6 +573,11 @@ public void run() {
                 eventLog.setId(UUIDUtils.randomUUID());
                 eventLog.setName(name);
                 eventLog.setProperties(properties);
+                TransmissionTarget aTransmissionTarget = (transmissionTarget == null) ? mDefaultTransmissionTarget : transmissionTarget;
+                if (aTransmissionTarget != null) {
+                    eventLog.addTransmissionTarget(aTransmissionTarget.mTransmissionTargetToken);
+                    // TODO add multiple targets
```

Comment:
what is this todo exactly?

Reply:
This is for the next release/feature. We have this in iOS, too.